README.md

```diff
@@ -1,24 +1,158 @@
 
-* Configuration
+> An Api only app which fetches random greetings from a database
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/mumo-esther/hello-react-front-end">React(Redux)</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li><a href="https://github.com/mumo-esther/hello-rails-back-end">Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- **Fetch random greeting**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need: Configure your code editor with HTML , CSS & JS and some other important extensions
+
+### Setup
+
+```
+git clone git@github.com:mumo-esther/hello-rails-back-end.git
+```
+
+### Install
+
+Install this project with:
+
+```
+cd hello-rails-back-end
+
+bundle install
+
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  rails s
+```
+
+### Run tests
+
+- Not available for now
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Esther Mawioo 💻**
+
+- GitHub: [@mumo-esther](https://github.com/mumo-esther/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add more api routes**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please clone it and try it. I know you're going to love it
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I'd want to thank Microverse(staffs , mentors , reviewers) for giving me the knowledge to build an amazing project like this.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I fork the project and make a contribution?**
+
+  Of course you can! First fork it and contribute to it.
+
+- **How should I ask a pull request**
+
+  - Step 1 : Click on the pull request button
+  - Step 2 : create pull request
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] 🚧 You did a great job documenting your project! 💯 However please make sure that you have a LICENSE and that its respective link in the README is working, you can follow these steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set up a LICENSE. Microverse suggests using MIT! 🤓 🔧